web/src/message_events.js

```diff
@@ -165,6 +166,7 @@ export function insert_new_messages(messages, sent_by_this_client) {
     stream_list.update_streams_sidebar();
     pm_list.update_private_messages();
     recent_topics_ui.process_messages(messages);
+    inbox_ui.update();
```

Comment:
I'm a bit worried that this will have a significant performance impact to be doing a full rerender every time we receive a new message; I guess I'll look at how the `inbox_ui` module works and see, as I've not gotten there yet.

Reply:
OK I've now read the code -- and it's at least careful to not do anything when the inbox view isn't visible, so probably wouldn't be a blocker for merging this, since it won't impact anyone not looking at that view.

Architecturally, I think probably the easiest way to make the live-update nice and performant will involve migrating to using SolidJS for this component, so I'm not sure it's worth digging deeply into that beyond doing profiling to see if the "slow" one gets is actually important. 

It may still make sense to pass information on what exactly we're changing, but it's not exactly hard to `git grep inbox_ui.update` and do an audit down the line, so I think this is not a blocker for merging it.

Reply:
Okay